src/features/pagination/js/pagination.js

```diff
@@ -368,8 +410,6 @@
           $scope.$on('$destroy', dataChangeDereg);
 
           var setShowing = function () {
-            $scope.showingLow = ((options.paginationCurrentPage - 1) * options.paginationPageSize) + 1;
-            $scope.showingHigh = Math.min(options.paginationCurrentPage * options.paginationPageSize, options.totalItems);
```

Comment:
Are showingHight and last page index really the same thing? If so, how did you come to that conclusion?